routes/users.js

```diff
@@ -242,11 +242,11 @@ router.post("/login", async (req, res) => {
 });
 
 // Register Handle
-router.post("/register", async (req, res) => {
-  const { name, email, password, location } = req.body;
-  console.log({ name, email, password, location })
 
-  //check required fields
+// Register Handle
+router.post("/register", async (req, res) => {
+  const { name,email,password,location} = req.body;
```

Comment:
for logs of server necessary to have information about creating of new users
can you add that console log here?

console.log({ name, email, password, location })

Reply:
fixed it